package.json

```diff
@@ -73,13 +76,12 @@
     ]
   },
   "scripts": {
-    "bootstrap:hoist": "lerna bootstrap --hoist --force-local",
     "clean": "npm run clean:root && npm run clean:lib",
     "clean:install": "rm package-lock.json; npm run clean && time npm install",
     "clean:lib": "rm -rf packages/**/lib",
     "clean:root": "$(lerna clean --yes || true) && echo 'removing ./node_modules' && rm -rf node_modules && echo 'removed ./node_modules'",
     "compile:prod": "rm -rf ./build; NODE_ENV=production webpack --mode=production --config webpack.dev-site.config.js",
-    "compile": "npm run clean:lib && lerna run compile --parallel",
+    "compile": "npm run clean:lib && lerna run compile",
```

Comment:
why is the parallel flag removed?

Reply:
@sdadn Parallel  is already a feature in lerna v6, so we don't really need to specify it.